src/components/TaskRunDetails.vue

```diff
@@ -60,16 +60,9 @@
 
     <p-key-value label="Version" :value="taskRun.taskVersion" :alternate="alternate" />
 
-    <div v-if="taskRun.empiricalPolicy?.maxRetries">
```

Comment:
This field has been deprecated in Core for ~2 years so I think we're safe to stop displaying it.